ccm_web/server/src/api/api.service.ts

```diff
@@ -3,13 +3,15 @@ import { Injectable } from '@nestjs/common'
 import { ConfigService } from '@nestjs/config'
 
 import { handleAPIError } from './api.utils'
-import { CanvasCourse, CanvasCourseBase, CanvasCourseSection } from '../canvas/canvas.interfaces'
+import { CanvasCourse, CanvasCourseBase, CanvasCourseSection, CanvasEnrollment } from '../canvas/canvas.interfaces'
 import { APIErrorData, Globals } from './api.interfaces'
 import { CreateSectionApiHandler } from './api.create.section.handler'
 import { CanvasService } from '../canvas/canvas.service'
 import { User } from '../user/user.model'
 
 import baseLogger from '../logger'
+import { EnrollSectionUsersApiHandler } from './api.enroll.section.users.handler'
+import { SectionUserDto } from './dtos/api.section.users.dto'
```

Comment:
Nit-pick: Group with other `api` imports?

Reply:
So, just move these lines up with lines 7–8?

Reply:
So, just move these lines up with lines 7–8?

Reply:
Yes. I'm trying not to be too particular about imports, since it's low on the "it matters" scale, but here's how I generally approach ordering them.

1) Library imports, alphabetical by package name
2) NestJS module imports, current directory and subdirectories first, then directories on higher levels, alphabetical by file and directories.
3) Stuff outside of Nest modules (like `logger`, `config`, etc.), similar file and directory orders.

Hard to be consistent always, but grouping stuff from the same directory works out pretty well with Nest module imports.